tests/test_airpyllution.py

```diff
@@ -15,119 +16,237 @@ def __init__(self, json_data, status_code):
+            mock_params["start"],
+            mock_params["end"],
+            mock_incorrect_params["lat"],
+            mock_params["lon"],
+            mock_params["appid"],
+        )
+        == "Latitude input should be a float"
+    )
+
     # Invalid input type
-    assert airpyllution.get_pollution_history(
-        mock_params['start'], 
-        mock_params['end'], 
-        mock_params['lat'], 
-        mock_incorrect_params['lon'], 
-        mock_params['appid']) == "Longitude input should be a float"
+    assert (
+        airpyllution.get_pollution_history(
+            mock_params["start"],
+            mock_params["end"],
+            mock_params["lat"],
+            mock_incorrect_params["lon"],
+            mock_params["appid"],
+        )
+        == "Longitude input should be a float"
+    )
 
     # Invalid API key
-    assert airpyllution.get_pollution_history(
-        mock_params['start'], 
-        mock_params['end'], 
-        mock_params['lat'], 
-        mock_params['lon'], 
-        mock_incorrect_params['appid']) == 'Invalid API key. Please see http://openweathermap.org/faq#error401 for more info.'
+    assert (
+        airpyllution.get_pollution_history(
+            mock_params["start"],
+            mock_params["end"],
+            mock_params["lat"],
+            mock_params["lon"],
+            mock_incorrect_params["appid"],
+        )
+        == "Invalid API key. Please see http://openweathermap.org/faq#error401 for more info."
+    )
 
     pollution_data_frame = airpyllution.get_pollution_history(
-        mock_params['start'], 
-        mock_params['end'], 
-        mock_params['lat'], 
-        mock_params['lon'], 
-        mock_params['appid'])
-    
+        mock_params["start"],
+        mock_params["end"],
+        mock_params["lat"],
+        mock_params["lon"],
+        mock_params["appid"],
+    )
+
     assert_frame_equal(pollution_data_frame, convert_data_to_pandas(mock_history_data))
 
-@patch('requests.get', side_effect=mocked_requests_get_pollution)
-def test_pollution_forecast(mock_api_call):
-    """Tests pollution forecast function"""
+
+@patch("requests.get", side_effect=mocked_requests_get_pollution)
+def test_air_pollution(mock_api_call):
+    """Tests get air pollution function"""
 
     # Invalid input type
-    assert airpyllution.get_pollution_forecast(
-        mock_incorrect_params['lat'],
-        mock_params['lon'],
-        mock_params['appid']) == "Latitude input should be a float"
-
-    assert airpyllution.get_pollution_forecast(
-        mock_params['lat'],
-        mock_incorrect_params['lon'],
-        mock_params['appid']) == "Longitude input should be a float"
-
-    assert airpyllution.get_pollution_forecast(
-        mock_params['lat'],
-        mock_params['lon'],
-        0) == "API Key should be a string"
-    
+    assert (
+        airpyllution.get_air_pollution(
+            mock_incorrect_params["lat"], mock_params["lon"], mock_params["appid"]
```

Comment:
Just a preference, is it possible to change your VSCode linter so that it doesn't put these parameters on a single line? I feel like it's more readable when they're on separate rows - dw if that's annoying to fix though it's not urgent.

Reply:
I think it's good for this time. But I would recommend if @danfke you could change the VS code settings cause it makes it hard to review the code because there is a lot of change in the code.

Reply:
Yah I will probably just turn it off